### PR TITLE
Exposes zipkin.log-level to control logging without rebuilding

### LINKED ISSUE #30: Updating log configuration from command line
To enable debug logs, we have to update logback.xml and rebuilding the code. We should be able to configure log levels through command line parameters.

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Before, we could only control logging by rebuilding. This special-cases
the "zipkin" category such that it can be controlled without
distributing files to the cluster or changing dependencies.

Note: this only special-cases the zipkin log category. Third party code
that logs outside the zipkin category will need to do something more
involved, possibly creating and distributing a log4j.properties file and
enabling it on the executors.

This is a port of work done in https://github.com/openzipkin/zipkin-dependencies/pull/62

Fixes #30

### CODE REVIEW

File: consumer/storage/src/main/java/zipkin/sparkstreaming/consumer/storage/StorageConsumer.java [hunk 1]
```diff
@@ -48,18 +49,18 @@ Logger log() { // Override for testing. Instance variables won't work as Logger
     try {
       get().asyncSpanConsumer().accept(list, blockingCallback);
       blockingCallback.get();
-      log().info("Wrote " + list.size() + " spans");
```
Comment (adriancole) [inline consumer/storage/src/main/java/zipkin/sparkstreaming/consumer/storage/StorageConsumer.java]: info was obnoxiously high :P

**Comment by adriancole** [comment]
cc @naoman @shakuzen @devinsba

File: pom.xml [hunk 2]
```diff
@@ -223,6 +217,18 @@
...
     <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-log4j12</artifactId>
+      <version>1.7.24</version>
+    </dependency>
+
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <version>1.2.17</version>
```
Comment (shakuzen) [inline pom.xml]: Is Spark stuck on log4j 1? Will log4j2 not work with it? I just ask because log4j 1 is EOL now (last release being 5 years ago).

**Comment by adriancole** [comment]
>
> Is Spark stuck on log4j 1? Will log4j2 not work with it? I just ask
> because log4j 1 is EOL now (last release being 5 years ago).
>
we are running spark 1.6 at the moment, and the issue to allow log4j2 is
still unresolved in spark https://issues.apache.org/jira/browse/SPARK-6305

I think you can achieve it via surgery (controlling the classpath and
library order of executors etc), but for now it is not an easy out.

**Pull request merged**